tx-source/site_core.php

```diff
@@ -1581,7 +1589,9 @@
 $Definition['This Month\'s Leaders'] = 'This Month\'s Leaders';
 $Definition['This setting loads the page before executing Javascript which can improve your SEO.<br><strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong>'] = 'This setting loads the page before executing Javascript which can improve your SEO.<br><strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong>';
 $Definition['This question\'s answer has been accepted.'] = 'This question\'s answer has been accepted.';
+$Definition['A follow-up email was already sent.'] = 'A follow-up email was already sent.';
 $Definition['This user has been verified as a non-spammer.'] = 'This user has been verified as a non-spammer.';
+$Definition['This user has disabled this notification preference.'] = 'This user has disabled this notification preference.';
```

Comment:
```suggestion
$Definition['The user has disabled this notification preference.'] = 'The user has disabled this notification preference.';
```

I don't like sentences that specify "this" twice.